docs/visual-studio-extensibility/nuget-api-in-visual-studio.md

```diff
@@ -1253,3 +1254,71 @@ This interface was primarily used by the ASP.NET team, to suggest that Javascrip
         Task<bool> IsRestoreCompleteAsync(CancellationToken token);
     }
 ```
+
+## IVsNuGetProjectUpdateEvents interface
+
+```cs
+/// <summary>
```

Comment:
```suggestion
    /// <summary>
```

I hope that's the same indent as the other lines. Edge/GitHub isn't allowing me to copy-paste whitespace 😞

Reply:
I edited it directly.